raw-svo.defs.lua

```diff
@@ -210,7 +210,10 @@ defs_data = pl.OrderedMap {}
       end
 
       return true
-    end}) -- added in xml w/ conf.gagbreath
+    end,
+    def = [[You are holding your breath.]],
+    ondef = function() return "" end, --workaround to make it show up in defense lists...
```

Comment:
It will gag the line, but not tick the breathing defense...
The returned string gets saved in def_def_list and that gets appended to the def name (see https://github.com/svof/svof/blob/master/raw-svo.defs.lua#L3134)